src/app/components/horse-books-listing/horse-book-list-item/horse-book-list-item.component.scss

```diff
@@ -21,19 +19,28 @@ mat-card{
 	text-align: middle;
 }
 .image img{
+	padding-top: 10%;
 	width: 292px;
 	max-height: 366px;
+	position: absolute;
+	top: 50%;
+	transform: translate(0, -50%);
 }
 .title{
 	text-overflow: ellipsis;
 	overflow: hidden;
 	white-space: nowrap;
-	display: inline-flex;
+	text-align: center;
 }
 
 
-@media only screen and (max-width: 679px) {
+@media only screen and (max-width: 696px) {
```

Comment:
This shows you changing the width that the single-column stuff kicks in at from 679 to 696.

Reply:
Per our discussion in Discord, the list is 1 column starting at 696px, so this is an appropriate change. Good catch!